samples/bot-tag-mention/csharp/README.md

```diff
@@ -0,0 +1,138 @@
+---
+page_type: sample
+description: This sample app demonstrates the use of tag mention funtionality in teams scope using Bot Framework.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- csharp
+extensions:
+ contentType: samples
+ createdDate: "09/06/2023 20:38:25 PM"
+urlFragment: officedev-microsoft-teams-samples-Tag-Mention-Bot-csharp
+---
+
+# Tag mention bot
+
+This sample app demonstrates the use of tag mention funtionality in teams scope using Bot Framework.
+
+## Included Features
+* Bots
+* Adaptive Cards
+* Teams Conversation Events
+
+## Interaction with bot
+![Tag-mention ](Images/Tag-mention-bot.gif)
+
+## Prerequisites
+
+- Microsoft Teams is installed and you have an account
+- [.NET SDK](https://dotnet.microsoft.com/download) version 6.0
+- [dev tunnel](https://learn.microsoft.com/en-us/azure/developer/dev-tunnels/get-started?tabs=windows) or [ngrok](https://ngrok.com/) latest version or equivalent tunnelling solution
+- Create tags within the team channel prior to utilizing the bot.
+
+## Setup
+
+> Note these instructions are for running the sample on your local machine, the tunnelling solution is required because
+the Teams service needs to call into the bot.
+
+1) Run ngrok - point to port 3978
+
+   ```bash
+   ngrok http 3978 --host-header="localhost:3978"
+   ```
+
+   Alternatively, you can also use the `dev tunnels`. Please follow [Create and host a dev tunnel](https://learn.microsoft.com/en-us/azure/developer/dev-tunnels/get-started?tabs=windows) and host the tunnel with anonymous user access command as shown below:
+
+   ```bash
+   devtunnel host -p 3978 --allow-anonymous
+   ```
+
+2) Setup for Bot
+
+   In Azure portal, create a [Azure Bot resource](https://docs.microsoft.com/azure/bot-service/bot-service-quickstart-registration).
+    - For bot handle, make up a name.
+    - Select "Use existing app registration" (Create the app registration in Azure Active Directory beforehand.)
+    - __*If you don't have an Azure account*__ create an [Azure free account here](https://azure.microsoft.com/free/)
+    
+   In the new Azure Bot resource in the Portal, 
+    - Refer to [Bot SSO Setup document](https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/main/samples/bot-conversation-sso-quickstart/BotSSOSetup.md).
+    - Ensure that you've [enabled the Teams Channel](https://learn.microsoft.com/azure/bot-service/channel-connect-teams?view=azure-bot-service-4.0)
+    - In Settings/Configuration/Messaging endpoint, enter the current `https` URL you were given by running the tunneling application. Append with the path `/api/messages`
```

Comment:
I can't see any steps to setup bot authentication and how to configure connection in azure bot.

Reply:
https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/main/samples/bot-conversation-sso-quickstart/BotSSOSetup.md


I have added the above link and it consists everything

Reply:
Changed the readme accordingly. Fixed